policyengine_us/variables/gov/states/me/tax/income/credits/fairness/property_tax_fairness_credit/me_property_tax_fairness_credit_veterans_cap.py

```diff
@@ -14,4 +14,5 @@ def formula(tax_unit, period, parameters):
             period
         ).gov.states.me.tax.income.credits.fairness.property_tax
         base_cap = tax_unit("me_property_tax_fairness_credit_base_cap", period)
-        return base_cap * p.veterans_matched
+        is_veteran = tax_unit.any(tax_unit.members("is_veteran", period))
```

Comment:
Filing units qualify if anyone is a veteran, including dependents? Please cite statute or form in issues or PRs to corroborate.
```suggestion
        has_veteran = tax_unit.any(tax_unit.members("is_veteran", period))
```